onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3681,16 +3708,39 @@ def aten_lu_unpack(
     raise NotImplementedError()
 
 
-def aten_mH(self: TensorType) -> TensorType:
+@torch_op("aten::mH")
+def aten_mH(self: TReal) -> TReal:
```

Comment:
Should we replace TReal with TRealOrUInt8 since Einsum can also accept UInt8?

Reply:
ok!

Reply:
Done